contracts/dex_stargate/Cargo.toml

```diff
@@ -28,19 +28,17 @@ overflow-checks = true
 
 [features]
 # for quicker tests, cargo test --lib
-# for more explicit tests, cargo test --features=backtraces
-backtraces = ["cosmwasm-std/backtraces"]
 library = []
 
 [dependencies]
 neutron-sdk = { workspace = true }
-cosmwasm-std = { version = "1.4.0", features = ["stargate"] }
-cw2 = "1.1.1"
+cosmwasm-std = { version = "2.0.4", features = ["stargate"] }
+cw2 = "2.0.0"
```

Comment:
please set the version to workspace for consistency (I think we're safe to do so?)

Reply:
I can see `cw2`, `cw-storage-plus` and `cosmwasm-schema` versions inconsistency across the project, some of them are still of `1.x` versions. since we're bumping cosmwasm to a new major version, please set them all to workspace

Reply:
Had the same idea. Will do.

Reply:
Regarding `1.x` versions: there is only one contract left (`balance-tracker`) with deps of such versions and, unfortunately, it is impossible to update cosmwasm deps until `astroport` updates its [cosmwasm-std](https://github.com/astroport-fi/astroport-core/blob/main/Cargo.toml#L20)